src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,11 @@
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        if (UserService.findByEmail(email) != null) {
```

Comment:
let's save this User who we get from `findByEmail()` method, into a local variable